2_base_and_sensor.ipynb

```diff
@@ -309,48 +575,90 @@
   },
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "Let's compare the state of the simulator (on Gazebo) with the displayed data (on RViz).\n",
+    "\n",
+    "![title](./imgs/2_rviz_topics.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "ja"
+   },
    "source": [
     "<div class=\"alert alert-block alert-info\">\n",
     "    <b>課題</b>\n",
     "    <p>HSRが搭載している各種センサ情報をrviz上で確認し、スクリーンショット、スクリーンキャストを表示してみましょう。</p>\n",
     "</div>"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "lang": "en"
+   },
+   "source": [
+    "<div class=\"alert alert-block alert-info\">\n",
+    "    <b>Task</b>\n",
+    "    <p>Confirm the sensor information mounted on HSR and take screenshots.</p>\n",
```

Comment:
"    <p>Check the information from the sensors mounted on the HSR and take screenshots.</p>\n",